test/functional/p2p_addr_relay.py

```diff
@@ -208,6 +225,69 @@ def blocksonly_mode_tests(self):
 
         self.nodes[0].disconnect_p2ps()
 
+    def rate_limit_tests(self):
+
+        for contype, tokens, no_relay in [("outbound-full-relay", 1001, False), ("block-relay-only", 0, True), ("inbound", 1, False)]:
+            self.log.info('Test rate limiting of addr processing for %s peers' % contype)
```

Comment:
nit: use f-string instead?

Reply:
I don't find those generally more readable.

Reply:
For more complex formatting I think f-strings are more readable, because they specify the value where it is inserted, but if there is only one value, I agree that doesn't win anything.

Reply:
Done.